src/main/java/org/broadinstitute/consent/http/resources/TDRResource.java

```diff
@@ -64,4 +86,45 @@ public Response getDatasetByIdentifier(@Auth AuthUser authUser, @PathParam("iden
             return createExceptionResponse(e);
         }
     }
+
+    @POST
+    @Consumes("application/json")
+    @Produces("application/json")
+    @Path("/dar/draft")
+    @PermitAll
+    public Response createDraftDataAccessRequest(
+        @Auth AuthUser authUser,
+        @Context UriInfo info,
+        @QueryParam("identifiers") String identifiers,
+        @QueryParam("projectTitle") String projectTitle) {
+      try {
+        // Ensure that the user is a registered DUOS and SAM user
+        User user = userService.findOrCreateUser(authUser);
+        if (Objects.isNull(identifiers) || identifiers.isBlank()) {
+          throw new BadRequestException("No dataset identifiers were provided");
+        } else {
+          List<String> identifierList = Arrays.stream(identifiers.split(","))
+                  .map(String::trim)
+                  .toList();
+          List<Integer> datasetIds = tdrService.getDatasetIdsByIdentifier(identifierList);
+        // Check that we were able to find a dataset id for all identifiers provided
+        if (identifierList.size() != datasetIds.size()) {
+            throw new BadRequestException("Invalid dataset identifiers were provided.");
+        }
+        DataAccessRequest newDar = new DataAccessRequest();
+        DataAccessRequestData data = new DataAccessRequestData();
+        String referenceId = UUID.randomUUID().toString();
+        newDar.setReferenceId(referenceId);
+        data.setReferenceId(referenceId);
+        data.setProjectTitle(projectTitle);
+        newDar.setData(data);
+        newDar.setDatasetIds(datasetIds);
+        DataAccessRequest result = darService.insertDraftDataAccessRequest(user, newDar);
+        URI uri = info.getRequestUriBuilder().path("/" + result.getReferenceId()).build();
```

Comment:
The URI here is a little confusing, but we need one to return a `created` response object. This builds one based on the current URL that the client used, which includes url parameters. This is used in the response headers like so:

```
 access-control-allow-credentials: true 
 access-control-allow-headers: authorization,content-type,accept,origin,x-app-id 
 access-control-allow-methods: GET,POST,PUT,PATCH,DELETE,OPTIONS,HEAD 
 access-control-allow-origin: * 
 access-control-max-age: 1728000 
 connection: Keep-Alive 
 content-length: 246 
 content-type: application/json 
 date: Mon,26 Sep 2022 18:21:47 GMT 
 keep-alive: timeout=5,max=100 
 location: https://local.broadinstitute.org:27443/api/tdr/dar/draft/6068d46c-2e66-4102-b8af-91a4ccc24643?identifiers=DUOS-000593%2C%20DUOS-000594&projectTitle=TDR%20Testing%204 
 server: Apache 
 strict-transport-security: max-age=86400; includeSubDomains 
 vary: Origin 
 x-content-type-options: nosniff 
 x-frame-options: SAMEORIGIN
```

Note the `location` header. In principle, what this should return is the actual DAR url. There should be some standardized code that we can write to do this sort of thing, but that's way outside the scope of this PR. Instead, we should manually reconstruct the correct dar url like so:

```
URI uri = info.getRequestUriBuilder().replacePath("").path("api/dar/v2/" + result.getReferenceId()).replaceQuery(null).build()
```